product_docs/docs/pem/9/certificates/index.mdx

```diff
@@ -39,23 +39,23 @@ SSLCertificateFile /path/to/your_domain_name.crt
 
-You can replace the PEM self-signed SSL certificates and keys with the trusted third-party CA certificates. For more information, see [Trusted third-party CA certificates and keys](/pem/latest/certificates/#trusted-third-party-ca-certificates-and-keys).
+You replace the PEM self-signed SSL certificates and keys with the trusted third-party CA certificates and keys. For more information, see [Trusted third-party CA certificates and keys](/pem/latest/certificates/#trusted-third-party-ca-certificates-and-keys).
 
 ### How PEM self-signed SSL certificates work
 
-The PEM server configuration script generates self-signed SSL certificates and key files for the PEM backend database server. The backend database server uses these certificates and keys to authenticate and encrypt the agent connections. Each certificate has an expiry date. You should regenerate the certificates when it nears expiration. For more information, see [Regenerating server self-signed SSL certificates](/pem/latest/certificates/replacing_ssl_certificates).
+The PEM server configuration script generates self-signed SSL certificate and key files for the PEM backend database server. The backend database server uses these certificates and keys to authenticate and encrypt the agent connections. Each certificate has an expiry date. You should regenerate the certificates when it nears expiration. For more information, see [Regenerating server self-signed SSL certificates](/pem/latest/certificates/replacing_ssl_certificates).
 
 The PEM agent connects to the PEM backend database server using the libpq interface, acting as a client of the backend database server. The agent self-signed SSL certificates and keys get generated during [agent registration](/pem/latest/registering_agent). PEM agent establishes the connection with the PEM backend database server using the self-signed SSL certificate and key files. 
 
 Each agent has a unique identifier and the agent certificates and keys have the corresponding identifier mentioned. Each certificate has an expiry date. You should regenerate the certificates when it nears expiration. For more information, see [Regenerating agent self-signed SSL certificates](/pem/latest/certificates/regenerating_agent_certificates).
 
-You can generate a common agent certificate and key file pair for all the PEM agents and use for connecting to PEM backend database server. For more information, see [Generate common agent certificate and key pair](/pem/lates/certificates/#generate-a-common-agent-certificate-and-key-pair)
+Generate a common agent certificate and key file pair for all the PEM agents and use for connecting to PEM backend database server. For more information, see [Generate common agent certificate and key pair](/pem/lates/certificates/#generate-a-common-agent-certificate-and-key-pair)
```

Comment:
```suggestion
If required, it is possible to use the same certificate for all agents rather than one certificate per agent. For more information, see [Generate common agent certificate and key pair](/pem/lates/certificates/#generate-a-common-agent-certificate-and-key-pair)
```